sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/albert-base-v2/albert-base-v2.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Don't we have HuggingFace DLCs that should have all these deps in them already?</p><p> </p><p>https://github.com/aws/deep-learning-containers/blob/master/huggingface/pytorch/training/docker/1.11/py3/cu113/Dockerfile.trcomp.gpu</p><p> </p><p>If not, you could file an issue or PR to get them updated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>These dependencies are for the Python kernel that executes the notebook code which essentially consists of data preparation before training. All the training dependencies are captured in the DLC as you mentioned. I do not believe the training compiler DLCs become notebook/studio kernels.</p>